Seymour/Matroid/Constructors/StandardRepresentation.lean

```diff
@@ -110,18 +110,19 @@ lemma StandardRepr.toVectorMatroid_indep_iff_submatrix'' [DivisionRing R] (S : S
     ∃ hI : I ⊆ S.X ∪ S.Y, LinearIndependent R (S.Bᵀ.uppendId.submatrix (Subtype.toSum ∘ hI.elem) id) := by
   simpa using S.toVectorMatroid_indep_iff_submatrix' I
 
-/-- Every vector matroid has a standard representation. -/
-lemma VectorMatroid.exists_standardRepr [DivisionRing R] (M : VectorMatroid α R) :
-    ∃ S : StandardRepr α R, S.toVectorMatroid = M := by
-  sorry
-
 /-- Every vector matroid has a standard representation whose rows are a given base. -/
 lemma VectorMatroid.exists_standardRepr_isBase [DivisionRing R] {B : Set α}
     (M : VectorMatroid α R) (hMB : M.toMatroid.IsBase B) :
-    ∃ S : StandardRepr α R, M.X = B ∧ S.toVectorMatroid = M := by
+    ∃ S : StandardRepr α R, S.X = B ∧ S.toVectorMatroid = M := by
```

Comment:
It should be noted that this PR also corrects `M.X` -> `S.X` as the intended definition - the previous statement is not true.